docs/tutorials/estimator.ipynb

```diff
@@ -0,0 +1,238 @@
+{
```

Comment:
<p>Since you mentioned "session" here, perhaps a few words about what it is? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>